packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -23,7 +23,7 @@ import '../project.dart';
 import 'android_sdk.dart';
 import 'android_studio.dart';
 
-const String gradleVersion = '4.10.2';
```

Comment:
this will break plugins, so this PR needs to go after https://github.com/flutter/flutter/pull/35217 is merged.

Reply:
You refer to this particular line change or the PR as a whole?